src/app/core/services/navigation.service.ts

```diff
@@ -57,6 +57,12 @@ const navigationService = {
     const lastSegment = pathname[pathname.length - 1];
     return lastSegment;
   },
+  replaceState(uuid: string | undefined): void {
+    const pathname = navigationService.history.location.pathname.split('/');
+    pathname[pathname.length - 1] = uuid ?? '';
```

Comment:
Add a try catch in order to handle if the pathname lenght was 0 or other unexpected case to avoid app crashes